commands/deployment_strategy.go

```diff
@@ -30,23 +31,31 @@ func (b *BlueGreenDeployCommandDeploymentStrategy) CreateProcessBuilder() *util.
 	processBuilder.Parameter("noConfirm", strconv.FormatBool(b.noConfirm))
 	processBuilder.Parameter("skipIdleStart", strconv.FormatBool(b.skipIdleStart))
 	processBuilder.Parameter("keepOriginalAppNamesAfterDeploy", strconv.FormatBool(true))
+	if b.rollingDeploy {
```

Comment:
Can we remove the if statement and just use it this way `processBuilder.Parameter("shouldApplyIncrementalInstancesUpdate", strconv.FormatBool(b.rollingDeploy))` ?

Reply:
done